test/tasks/upgrade/config.js

```diff
@@ -58,7 +58,8 @@ test.cb('upgrade:config', function(t) {
 		assert.fileContentMatch(pluginPackagePropertiesPath, /7\.0\.0\+/);
```

Comment:
Hey @blzaugg 

This assertion is failing, it's looking for `7_0_0` in the look and feel file.

```
<!DOCTYPE look-and-feel PUBLIC "-//Liferay//DTD Look and Feel 7.0.0//EN" "http://www.liferay.com/dtd/liferay-look-and-feel_7.0.0.dtd">
```

That's what the file's doctype is after the upgrade task is run.